roadmap.md

```diff
@@ -81,20 +82,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
 
 |Milestone|Release Date|
 |---------|------------|
-|RTM      |     Q1 2016|
+|1.0      |   June 2016|
+|1.1      |   Fall 2016|
+
+Planned 1.1 features
+--------------------
+
+- Broader API support, bringing parity with .NET Framework and Mono.
+- Transition to MSBuild and csproj as the default build system and project model for all versions of .NET. 
 
 Notes:
 
-- The RC1 release candidate will be a supported and production ready cross-platform release. 
-- Depending on feedback from RC1, we will ship additional release candidates as necessary.
-- CoreFX API contracts may need to be at RTM quality in RC releases, to support the already RTM Windows 10 UWP release.
-- The [ASP.NET 5 roadmap](https://github.com/aspnet/Home/wiki/Roadmap) articulates the ASP.NET projects's roadmap and dates.
+- The 1.0 release is accompanied with a preview version of the Visual Studio and command-line tooling. The tooling should reach RTM quality with version 1.1 of the .NET Core runtime in Fall 2016.
+- The [ASP.NET Core 1.0 roadmap](https://github.com/aspnet/Home/wiki/Roadmap) articulates the ASP.NET projects's roadmap and dates.
 
 TBD
 ===
 
 There are several areas of the product that are TBD. This designation doesn't mean that they are not important, just that there is no plan in place yet. Feel free to contribute that plan.
 
 - OS package manager support (e.g. apt-get, brew).
-- Broader Linux distro support (beyond Debian and CentOS).
+- Broader Linux distro support.
```

Comment:
We should link these two points: OS package manager support and broader distro support. They are not distinct.